components/Rank/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {mount} from 'enzyme'
+import React from 'react'
+import Rank from '../'
+
+describe('Rank', () => {
+  beforeEach(() => {
+    console.error = jest.fn((error) => {
```

Comment:
I'm not happy with this overwriting.  I imagine that ` console.error ` will remain mocked.  Moving it to a global test bootstrap would be 1 option.  Good to hear peoples opinions of this.